paypalscript.py

```diff
@@ -40,12 +52,17 @@ def write(self):
                                                  'Defaults output file name is Output.csv')
     parser.add_argument('-i', '--input', default='Paypal Export.csv', help='Input file name')
     parser.add_argument('-o', '--output', default='Output.csv', help='Output file name')
+    parser.add_argument('-d', '--date', required=True, help='Ignore entries including and before this date ('
+                                                            'mm/dd/yyyy hh/mm/ss)',
+                        type=lambda s: datetime.strptime(s, "%m/%d/%Y %H:%M:%S"))
```

Comment:
Just a UX suggestion, change this to a default, say one month before the current execution time, rather than a required date that they have to go look up

Reply:
Recap from conversation, going to skip this recommendation due to needing the user to specify the date from the last GnuCash entry to avoid creating duplicate entries, which GnuCash will happily accept.